metafields.go

```diff
@@ -7,6 +7,27 @@ import (
 	"time"
 )
 
+// ConvertToInt converts strings, float64 or int64 to int
+func ConvertToInt64(v any) (int64, error) {
+	switch value := any(v).(type) {
+	case int64:
+		return value, nil
+	case float64:
+		return int64(value), nil
+	case int:
+		return int64(value), nil
+	case string:
+		converted, err := strconv.ParseInt(v.(string), 0, 64)
+		if err != nil {
+			fmt.Println("Error parsing string to int:", err)
+			return 0, err
+		}
+		return converted, nil
+	default:
+		return 0, fmt.Errorf("unsupported type %T", v)
+	}
+}
```

Comment:
Is there anywhere else that this might want to be used? Seems like a nice generic helper function.

Reply:
It does, although it's only used in this file 🙂 if we require it somewhere else, we can move it into a helper function 👌